lib/perl/Genome/Model/Build/ReferenceSequence/Converter.pm

```diff
@@ -57,11 +57,20 @@ sub __errors__ {
     return @errors;
 }
 
+sub exists_for_references {
+    my $class = shift;
+    my $source_reference = shift;
+    my $destination_reference = shift;
+
+    my @ref = $class->_faster_get(source_reference_build => $source_reference, destination_reference_build => $destination_reference);
```

Comment:
For a separate PR, maybe we should rename `_faster_get()` to `lookup()`?  Doesn't seem like there's any reason why this method needs to be "private".